### DISCUSSION TITLE
What are these three files for?

### DISCUSSION THREAD

**Discussion by derpthawt** [Open-Shell/Open-Shell-Menu#2010] [Q&A] [answered]
What are these three files for?  

OpenShellSetup_4_4_190.exe  

OpenShellSymbols_4_4_190.7z  

Utility.exe  

Do I need all three to install in WIndows11, or just the setup file?

**Comment by ge0rdi** [comment, marked as answer]
> Do I need all three to install in WIndows11, or just the setup file?

You only need to run the _setup_.

> What are these three files for?

`OpenShellSymbols_*.7z` are symbol files used for debugging/crash dump analysis.
`Utility.exe` is our tool that does various trouble-shooting tasks.

**Reply by derpthawt** [reply]
Thank you!

**Reply by GitttyHubby** [reply]
Perhaps you could include this information in the readme.md?

**Reply by LinuxOnTheDesktop** [reply]
I think that that inclusion would be a good idea.
